sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/BlobServiceSasModelsTest.groovy

```diff
@@ -236,25 +256,4 @@ class BlobServiceSasModelsTest extends Specification {
         "container"  | "blob"  | "snapshot"  | null      | new BlobSasPermission().setReadPermission(true)                                  || "bs"     | "r"
         "container"  | "blob"  | null        | "version" | new BlobSasPermission().setReadPermission(true)                                  || "bv"     | "r"
     }
-
-    @Unroll
-    def "ensure state permission illegal argument"() {
```

Comment:
This test no longer makes sense since Blob and Container SAS have same permissions now